src/shared/utils/env.ts

```diff
@@ -0,0 +1,13 @@
+export type ENV = string
+
+/**
+ * Type guard to check whether a value is an ENV
+ **/
+export const isEnv = (env: ENV | undefined): env is ENV => !!env
+
+/**
+ * Returns a given ENV if it's valid only or returns a default value
+ * @param env {string} ENV
+ * @param defaultValue {string} Default value
+ */
+export const envOrDefault = (env: ENV | undefined, defaultValue: string) => (isEnv(env) ? env : defaultValue)
```

Comment:
All copied from `src/renderer/helpers/envHelper.ts` - #1739 will remove sources in `common` later